### ISSUE TITLE
BEE2.4.0 Release Checklist

### ISSUE THREAD

**Issue by Luke18033** [BEEmod/BEE2-items#1804]
This is a checklist of all the features that must be implemented before we can release the full BEE2.4.0 (Release 1).

- [ ] Finish BTS style
  - [ ] Corridors
  - [ ] Styled items
  - [ ] Make Co-op actually work
- [ ] Fix most bugs - as of this writing we have 13 pages of bugs, we need to get that down to 1-3 pages.
- [ ] Reimplement BEE2.2/Addons items
- [ ] Implement TWTM compatibility and style
- [ ] Implement missing styles (does not include custom)
  - Higher Priority
    - [ ] Art Therapy
    - [ ] Refurbished
    - [ ] Wheatley
    - [ ] Reconstructing
  - Lower Priority
    - [ ] Portal 1 BTS
    - [ ] Old Aperture BTS
    - [ ] Overgrown BTS
    - [ ] Aperture Tag Stage
- [ ] Improve the drop-down bar to support new features (BEEmod/BEE2.4#212).
- [ ] Re-add Mac support

Anything I've missed?

**Comment by LautaroL20** [comment]
It´d be better to call it Final Release. Things like the additions of New Styles like "P1 BTS", "Over BTS" Etc... Can be actually added at a later date without detracting the "Final Release" adjective (See Portal 2 and their DLCS that have been added later).

**Comment by LautaroL20** [comment]
And of course that the "Dropdown Bar" must be higly improved for the final release, see this: https://github.com/BEEmod/BEE2.4/issues/212

**Comment by Luke18033** [comment]
No. Later on we will make BEE2.4.2, BEE2.4.3, etc.

**Comment by Luke18033** [comment]
BEE2.4.1 only needs to include styles that appear in the campaign, Portal 1, or mods that are/will be supported. Many other styles have been suggested such as Prime 1950s or Blue Portals, which don't need to be included in this release.

**Comment by Luke18033** [comment]
Just updated the Epic with the issue you mentioned

**Comment by ZombiePwner1** [comment]
@Luke18033 Art therapy is probably gonna take quite awhile to implement since @TeamSpen210 Doesn't have any ideas on how the elevators should look like.

**Comment by Luke18033** [comment]
Refurbished as well, currently it's just Clean with Old Aperture textures.

**Comment by Luke18033** [comment]
For release versions like this, maybe we should include the default packages with the program.

**Comment by PorTalZ101** [comment]
Can't wait till you finish this your ideas look good

**Comment by Luke18033** [comment]
The first version would be 2.4.0, not 2.4.1, I've renamed it.

**Comment by Luke18033** [comment]
No longer relevant, the versioning system has changed.

**Comment by LautaroL20** [comment]
@Luke18033 - Do not forget to apply the label after closing an issue; since although it may sounds unbelievable, there are some people who still checks inside the "closed issues" tab in order to see which issues were solved/closed and their reasons. So, having them shorted with the proper label after closing will help people to understand the real cause of its closure.

There is no need to apply aditional labels such as "enhancement", "new item" etc, when the issue is invalid/duplicate. The "invalid" or "duplicate" labels are just enough to clarify a reason for it.

**Comment by Luke18033** [comment]
I didn't really consider it "invalid"? It definitely wasn't when I opened this issue a year and a half ago, I only closed it because of the new versioning.
